utils/RequestUtil.js

```diff
@@ -71,13 +71,10 @@ var RequestUtil = {
   },
 
   json: function (options) {
+    var usingHangingRequest = Util.isFunction(options.hangingRequestCallback);
     if (options) {
```

Comment:
Do we need this? We are assuming `options` is defined in the line above this check.